perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark_test.py

```diff
@@ -133,7 +133,7 @@ def test_perform_testing_flat(self, mock_log, mock_record_time_of_operation,
     test_type = "flat"
     num_samples = 4
     results = {}
-    mount_flags = "--config-file=/tmp/config.yml  --implicit-dirs --rename-dir-limit=1000000 --stackdriver-export-interval=30s"
+    mount_flags = "--config-file=/tmp/config.yml --log-severity=trace --log-format \"text\" --log-file /tmp/gcsfuse-logs-rename.txt  --implicit-dirs --rename-dir-limit=1000000 --stackdriver-export-interval=30s"
```

Comment:
Does `"text"` really need those quotes around it?

Reply:
This line is obsolete but made similar change in common flags in build script.